cmd/promxy/main.go

```diff
@@ -154,10 +155,16 @@ func main() {
 	}
 	logrus.SetLevel(level)
 
-	// Set the log format to have a reasonable timestamp
-	formatter := &logrus.TextFormatter{
-		FullTimestamp: true,
+	var formatter logrus.Formatter
+	if opts.LogFormat == "json" {
```

Comment:
nit: I'd change this to a `switch` just because it seems clearer -- but definitely not a major issue :)